src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -192,18 +191,36 @@ class JesBackend extends Backend with LazyLogging {
     }
   }
 
+  /**
+   * Creates a set of JES inputs for a backend call.
+   */
   def generateJesInputs(backendCall: BackendCall): Iterable[JesInput] = {
     val adjustedPaths = adjustInputPaths(backendCall.call, backendCall.locallyQualifiedInputs)
-    def mkInput(fileTag: String, location: WdlFile) = JesInput(fileTag, backendCall.lookupFunction(fileTag).valueString, Paths.get(location.valueString))
+    def mkInput(locallyQualifiedTaskName: String, location: WdlFile) = JesInput(locallyQualifiedTaskName, backendCall.lookupFunction(locallyQualifiedTaskName).valueString, Paths.get(location.valueString))
```

Comment:
should probably be `locallyQualifiedInputName` because it's not the name of a task